packages/squiggle-lang/src/rescript/Utility/SafeFloat.res

```diff
@@ -0,0 +1,34 @@
+type finite = Finite(float)
+module Finite = {
+  type t = finite
+  let valid = Js.Float.isFinite
+  let make = (x: float) : option<t> =>
```

Comment:
Is the idea that these would later be results, with error messages if it's not finite? Or to stay like this?